src/sentry/api/endpoints/project_triage_stats.py

```diff
@@ -0,0 +1,94 @@
+from __future__ import absolute_import
+
+from rest_framework.response import Response
+
+from datetime import timedelta
+from django.utils import timezone
+
+from sentry.api.bases.project import ProjectEndpoint
+from sentry.models import Group, GroupStatus, Activity
+from sentry.utils.dates import to_timestamp
+
+
+class TriageStatus(GroupStatus):
+    DNE = -1
+    ASSIGNED = 6
+
+
+class ProjectTriageStatsEndpoint(ProjectEndpoint):
+
+    def filter_activity_set(self, set, cutoff):
+        return [ai for ai in set if ai['datetime'] < cutoff]
+
+    def process_activity_set(self, group, dates):
+
+        activity_set = Activity.objects.using('default_analytics').filter(
```

Comment:
You need to coerce this to a list too. Gotta be careful with QuerySets.  It's going to end up being evaluated on L39 when you do `activity_set[seen_activities:]` on every date by applying an `OFFSET n` to the SQL.

Reply:
good lord

Reply:
nice catch